docs/docs/faq/question_05.md

```diff
@@ -36,4 +36,26 @@ with open("/path/to/my_inference.pkl", "wb") as handle:
 ```
 However, saving and loading the `inference` object will slightly modify the object (in order to make it serializable). These modifications lead to the following two changes in behaviour:
 1) Retraining from scratch is not supported, i.e. `.train(..., retrain_from_scratch=True)` does not work.
-2) When the loaded object calls the `.train()` method, it generates a new tensorboard summary writer (instead of appending to the current one).
\ No newline at end of file
+2) When the loaded object calls the `.train()` method, it generates a new tensorboard summary writer (instead of appending to the current one).
+
+## I trained a model on a GPU. Can I load it on a CPU?
+
+Yes. This can be useful for quick inspection and plotting. Currently, resuming training on CPU for an inference object trained on a GPU is not supported. If this is strictly required by your workflow, consider setting `inference._device = "cpu"` before calling `inference.train()`.
+
+```python
```

Comment:
Maybe I am missing something, but how is this code snippet related to the answer to the question above? 

I guess it is a workaround to still be able to load and inspect a net on the CPU, although it was trained on the GPU? 
If yes, I think it would be good to add a short comment above the snippet.

Reply:
Good point, thanks! There really are two separate points here:
1. one can load objects on CPU (main code snippet)
2. if you want to keep training, then there is a suggested workaround, but this is not supported.


Clarified this in new commit

Reply:
makes sense now, thanks!